src/tmx/TmxUtils/CMakeLists.txt

```diff
@@ -33,6 +34,7 @@ TARGET_LINK_LIBRARIES (${PROJECT_NAME} PUBLIC
                        rdkafka++ 
                        ::carma-clock
                        gmock
+                       jsoncpp
```

Comment:
Is this still necessary. No right since we are using boost property tree.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/649/commits/8ecd06c6f047aef1f29efa18e5d717212d19b3ea